addons/medical_treatment/initSettings.sqf

```diff
@@ -217,11 +207,11 @@
 ] call CBA_settings_fnc_init;
 
 [
-    QGVAR(treatmentTimeStich),
+    QGVAR(woundStitchTime),
```

Comment:
Different since this controls the amount of time to stitch a single wound. I thought this name implied that better but can revert if we want to keep the same format.

Reply:
it makes sense, I just wanted to double check